modules/apps/transfer/types/trace.go

```diff
@@ -70,6 +74,22 @@ func (dt DenomTrace) GetFullDenomPath() string {
 	return dt.GetPrefix() + dt.BaseDenom
 }
 
+func extractPathAndBaseDenomFromDenomTrace(denomTraceItems []string) (string, string) {
+	path := []string{}
+	baseDenom := []string{}
```

Comment:
more nits on declare vs assign, personally I prefer declare but its minor anyways, maybe we could benefit from a style guide at some point! 

```suggestion
	var path []string
	var baseDenom []string
```